src/core/error.js

```diff
@@ -37,15 +37,14 @@ export function duplicateErrorIfNecessary(error) {
 }
 
 /**
- * @param {...*} var_args
+ * @param {...*} args
  * @return {!Error}
  * @visibleForTesting
  */
-export function createErrorVargs(var_args) {
+export function createErrorVargs(...args) {
   let error = null;
   let message = '';
-  for (let i = 0; i < arguments.length; i++) {
-    const arg = arguments[i];
+  for (const arg of args) {
```

Comment:
Just iterate `arguments`.

Reply:
Done, removed spread changes in this file.